cpp/foxglove-websocket/include/foxglove/websocket/common.hpp

```diff
@@ -13,6 +14,12 @@ constexpr char CAPABILITY_TIME[] = "time";
 constexpr char CAPABILITY_PARAMETERS[] = "parameters";
 constexpr char CAPABILITY_PARAMETERS_SUBSCRIBE[] = "parametersSubscribe";
 constexpr char CAPABILITY_SERVICES[] = "services";
+constexpr char CAPABILITY_CONNECTION_GRAPH[] = "connectionGraph";
+
+constexpr std::array<const char*, 5> DEFAULT_CAPABILITIES = {
```

Comment:
I think we should be able to omit this and not have to worry about getting the number right?
```suggestion
constexpr std::array DEFAULT_CAPABILITIES = {
```

Reply:
You are right, this is much better